### PR TITLE
Change groups city column to address

### PR DESCRIPTION
**Pull request by blamb888**
20210325033949_rename_city_to_address_in_groups rb - rpg-seznamka  WSL Ubuntu  - Visual Studio Code
schema rb - rpg-seznamka  WSL Ubuntu  - Visual Studio Code
group rb - rpg-seznamka  WSL Ubuntu  - Visual Studio Code

### CODE REVIEW

**Review by Naokimi — CHANGES_REQUESTED** [review]
testing fails
raid why and update tests accordingly

**Comment by blamb888** [comment]
image

**Comment by blamb888** [comment]
schema rb - rpg-seznamka  WSL Ubuntu  - Visual Studio Code
group rb - rpg-seznamka  WSL Ubuntu  - Visual Studio Code
group_spec rb - rpg-seznamka  WSL Ubuntu  - Visual Studio Code

File: db/migrate/20210325033949_rename_city_to_address_in_groups.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RenameCityToAddressInGroups < ActiveRecord::Migration[6.1]
```
Comment (Naokimi) [inline db/migrate/20210325033949_rename_city_to_address_in_groups.rb]: remove this migration

File: db/migrate/20210402035233_rename_address_to_train_station_in_groups.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RenameAddressToTrainStationInGroups < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :groups, :address, :train_station
```
Comment (Naokimi) [inline db/migrate/20210402035233_rename_address_to_train_station_in_groups.rb]: change from address to city

**Review by Naokimi — CHANGES_REQUESTED** [review]
you still need to update the factory

File: db/migrate/20210325033949_rename_city_to_address_in_groups.rb (hunk 1 above)
Comment (blamb888) [reply db/migrate/20210325033949_rename_city_to_address_in_groups.rb:1]: If I remove that I'll need to change the new migration to

Comment (blamb888) [reply db/migrate/20210325033949_rename_city_to_address_in_groups.rb]: If I remove that migration I'll need to change the new migration to ```rename_column :groups, :city, :train_station```

File: db/migrate/20210402035233_rename_address_to_train_station_in_groups.rb (hunk 2 above)
Comment (blamb888) [reply db/migrate/20210402035233_rename_address_to_train_station_in_groups.rb]: Okay, I'll do that:)

**Comment by blamb888** [comment]
image
Updated factory from city to train_station.

**Comment by blamb888** [comment]
I FINALLY PASSED ALL THE TESTS!!!! Huzzah! I think I'm getting the hang of this now:)
image

**Review by Naokimi — APPROVED** [review]

**Pull request merged**